giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/users/UsersManager.java

```diff
@@ -76,6 +89,16 @@ public User findUserByEmail(String email) {
     @Override
     public User addUser(User user) throws UnstorableObjectException {
         client.addUser(user);
+        //send notification to all admins
+        List<User> adminList = client.getUsersByRole(GilesRole.ROLE_ADMIN.name());
+        for(User admin : adminList) {
+            try {
+                emailManager.sendAccountCreatedEmail(user.getName(), user.getUsername(), admin.getFullname(), admin.getEmail());
+            } catch (GilesNotificationException e) {
+                // let's log error but keep on going
+                messageHandler.handleMessage("Email to " + admin.getUsername() + " could not be sent.", e, MessageType.ERROR);
```

Comment:
let's make that a warning. It doesn't crash the system or anything, but could happen.